pages/10.users/02.access-control/docs.md

```diff
@@ -42,15 +42,15 @@ In your code, access is controlled through the use of access checks on permissio
 This can be done by calling the `checkAccess` method of the `AuthorizationManager` service. For example:
 
 ```php
-#[Inject]
+#[\DI\Attribute\Inject]
```

Comment:
I'm fine with using `#[\DI\Attribute\Inject]` instead of `#[Inject]` if it can help avoid the same issue you encounter (Attributes are new, and it's easy to miss they obey the import declaration) , but just an FYI  it's not necessary to use the full namespace if `use DI\Attribute\Inject` is at the top of the file.

Reply:
I chose this method just to match the previous #[Inject] sample in Chapter 5.
If we want to demonstrate alternate methods, it might be worth reverting this to `#[Inject]` but adding a `use` statement at the top.
Ping me if you'd like to see that and I'll submit a new PR :slightly_smiling_face:

Reply:
No you're right, it's either this, or add the "use" statement. I prefer this version for now. Thanks !